### PR TITLE
526 fix email validations mg

### PR DESCRIPTION
**Pull request by markgreenburg**
We were defining the email and phone schema for the 526 form in vets-website directly instead of using the definitions from vets-json-schema, which match swagger. This change fixes that.

### CODE REVIEW

File: src/applications/disability-benefits/526EZ/pages/primaryAddress.js [hunk 1]
```diff
@@ -168,7 +173,7 @@ export const uiSchema = {
       emailAddress: {
         'ui:title': 'Email address',
         'ui:errorMessages': {
-          pattern: 'Please put your email in this format x@x.xxx'
+          pattern: 'Email format must be at least x@x.xx'
```
Comment (cvalarida) [inline src/applications/disability-benefits/526EZ/pages/primaryAddress.js]: This could use some revision.

**Review by cvalarida — COMMENTED** [review]
Looks good minus a weird error message.

File: src/applications/disability-benefits/526EZ/pages/primaryAddress.js (hunk 1 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/pages/primaryAddress.js]: For reference, the actual validation is that the email must have a minimum of `x@x.xx`

Comment (markgreenburg) [reply src/applications/disability-benefits/526EZ/pages/primaryAddress.js]: Sure, thanks @peggygannon !

**Review by cvalarida — APPROVED** [review]

**Pull request merged**